src/control/actor/prometheus/start.go

```diff
@@ -182,13 +201,29 @@ func (c *PrometheusStartCmd) Run(ctx context.Context, args ...string) error {
 						unk += 1
 					}
 				}
+				// Client Version Distribution
+				client, version := export.FilterClientType(p.ClientType)
+				clientVer := fmt.Sprintf("%v_%v",client,version)
+				_, ok := clientVerDist[clientVer]
+				if ok {
+					clientVerDist[clientVer] += 1
+				} else {
+					clientVerDist[clientVer] = 1
+				}
 				// Generate the Country Code distribution
-				_, ok := geoDist[p.CountryCode]
+				_, ok = geoDist[p.CountryCode]
 				if ok {
 					geoDist[p.CountryCode] += 1
 				} else {
 					geoDist[p.CountryCode] = 1
 				}
+				// Generate the IP Address distribution
+				_, ok = ipDist[p.Ip]
```

Comment:
Cool metric. Just curious, have you seen this happening? If so, how many peers were running from the same ip?

Reply:
This is the distribution I got after ~1h30 running the crawler locally on my machine
![local_test_dashboard_2](https://user-images.githubusercontent.com/52137069/128685503-699a4ded-5d5f-466c-9976-c208fdde84dd.png)

Reply:
Interesting. Note sure how to interpret the data though. I mean, 2 ips contained 55 peers? or 55 ips contained 2 peers?

Reply:
The latter, 55 ips contained 2 peers. I'm sorry I did the graph to show it to you but I forgot to add a legend

Reply:
Cool thanks, kind of imagined it was like that :) Isn't it weird that we don't have x ips with 1 peer? Which should be the most common scenario?

Reply:
Yes It is, but I excluded them from the graph because there isn't an option for logarithmic graphs on grafana